rolemanager/role_manager.go

```diff
@@ -741,6 +741,15 @@ func (b *base) isValidPermission(permission string) bool {
 	return valid
 }
 
+func (b *base) IsValidPermissions(permissions gimlet.Permissions) error {
+	for permission := range permissions {
+		if !b.isValidPermission(permission) {
+			return errors.Errorf("'%s' is not a valid permission", permission)
+		}
+	}
```

Comment:
does it make sense to make this a catcher with continue on error semantics so that you don't have to iteratively find errors with submitted permissions

Reply:
Yep, changed